.github/workflows/cuda.yml

```diff
@@ -4,88 +4,21 @@ on:
   workflow_dispatch:
 
 jobs:
-  start-runner:
-    name: Start self-hosted EC2 runner
-    runs-on: ubuntu-latest
-    outputs:
-      label: ${{ steps.start-ec2-runner.outputs.label }}
-      ec2-instance-id: ${{ steps.start-ec2-runner.outputs.ec2-instance-id }}
-    steps:
-      - name: Configure AWS credentials
-        uses: aws-actions/configure-aws-credentials@v4
-        with:
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          aws-region: us-east-2
-      - name: Start EC2 runner
-        id: start-ec2-runner
-        uses: machulav/ec2-github-runner@v2
-        with:
-          mode: start
-          github-token: ${{ secrets.PAT }}
-          ec2-image-id: ami-04cd9fec4a7a39019
-          ec2-instance-type: g4dn.xlarge
-          subnet-id: subnet-72d3e53e
-          security-group-id: sg-06b0c93122c08aeab
-
   test:
-    name: Do the job on the runner
-    needs: start-runner # required to start the main job when the runner is ready
-    runs-on: ${{ needs.start-runner.outputs.label }} # run the job on the newly created runner
+    name: Test on CUDA Build
+    runs-on: nvidia
     container:
       image: ghcr.io/deepmodeling/abacus-cuda
       options: --gpus all
     steps:
       - name: Checkout
         uses: actions/checkout@v4
-      - name: Build cuSolver
+        with:
+          submodules: recursive
+      - name: Build
         run: |
           nvidia-smi
-          cmake -B build -DUSE_CUSOLVER_LCAO=ON
+          cmake -B build -DUSE_CUDA=ON -DBUILD_TESTING=ON
```

Comment:
@dzzz2001 Are there any other compilation options that need to be added to currently control the use of `cusolver` in LCAO basis?

Reply:
No, -DUSE_CUDA=ON is enough.